lib/sufia/import/generic_file_translator.rb

```diff
@@ -1,32 +1,20 @@
 module Sufia::Import
   # Imports a Sufia 6.0-exported GenericFile into a Sufia PCDM GernericWork and FileSet
-  class GenericFileTranslator
-    # @param Hash settings
+  class GenericFileTranslator < Translator
+    # @param Hash settings (see super for more)
     #   @attr import_binary    - (default true) Import the binary content from sufia6 fedora instance
     #     If true, sufia6_user and sufia6_password must be set in the appropriate sections of fedora.yml
     def initialize(settings)
+      super
+      @filename_prefix = settings.fetch(:filename_prefix, "generic_file_")
```

Comment:
I like this idea! Will give it a try.

having trouble with rubocop in the meantime; for some reason I'm not able to pull the new version locally.